nix/common/default.nix

```diff
@@ -0,0 +1,57 @@
+# This should be applied on all hosts
+{ config, lib, pkgs, modules, sources, nixpkgs, ... }: {
+  imports = [
+    # Set options intended for a "small" NixOS: Do not build X and docs.
+    (nixpkgs + "/nixos/modules/profiles/minimal.nix")
+
+    ./users.nix
+    ./sops.nix
+  ];
+
+  # Keep log file disk usage in check.
+  # The default is 10% of the partitition size or so.
+  services.journald.extraConfig = ''
+    # One week of logs ought to be enough
+    MaxRetentionSec=${toString(7 * (24 * 60 * 60))}s
```

Comment:
you can say `1week` here instead of the complicated `${toString(7 * (24 * 60 * 60))}s` expression